### ISSUE TITLE
Can attack even though all nodes are disabled

### ISSUE THREAD

**Issue by IceRaptor** [BattletechModders/CBTBehaviorsEnhanced#74]
Per multiple tickets, melee can happen even if all selections are disabled somehow

```
Ive had this happen to me a couple times now where an enemy melees me and does no damage. Im not sure if that means theyre trying to do an attack they dont have?

Harkonnen11/05/2020
@FrostRaptor 
07:17:03.002  - valid attacks => charge: False  dfa: False  kick: False  weapon: False  punch: False
7:17:04.089 Melee sequence 1140 created for attacker: Wolfhound WLF-1A_Recruit_199E9B50 vs. target: Toro TR-A-1_Swamp_57245CBC using
position: (-192.0, 188.9, 457.3)
and yet it tries to attack

Jaroth11/05/2020
Neat. I was right. I noticed it a couple times yesterday, that was the one I was able to catch for sure though because the toro was at full armor.
I didn’t think there was a way to have 0 melee options though?
```

**Comment by IceRaptor** [comment]
[RogueLogs(1).zip](https://github.com/BattletechModders/CBTBehaviorsEnhanced/files/5506787/RogueLogs.1.zip)

**Comment by IceRaptor** [comment]
[RogueLogs(2).zip](https://github.com/BattletechModders/CBTBehaviorsEnhanced/files/5506788/RogueLogs.2.zip)

**Comment by IceRaptor** [comment]
[RogueLogs(3).zip](https://github.com/BattletechModders/CBTBehaviorsEnhanced/files/5506791/RogueLogs.3.zip)

**Comment by IceRaptor** [comment]
[RogueLogs_OldMelee.zip](https://github.com/BattletechModders/CBTBehaviorsEnhanced/files/5506805/RogueLogs_OldMelee.zip)

**Comment by IceRaptor** [comment]
Fixed in 0.5.27
